:bookmark: Release 2.7.0 (#382)
maykinmedia/open-klant: CHANGELOG.rst

Change history
==============

2.7.0
=====
*April, 1, 2025*

**New features**

> * [:open-klant:`212`] Expand BetrokkeneSerializer with digitale_adressen

Reviewer: ```suggestion
* [:open-klant:`212`] Add ``digitaleAdressen`` as expand option for ``/betrokkenen`` endpoint
```

* [:open-klant:`239`] Accept Partij Identificatoren as part of Partij creation

> * [:open-klant:`355`] Add Notifications for InterneTaak and Partij

Reviewer: ```suggestion
* [:open-klant:`355`] Add Notifications for InterneTaak and Partij (see :ref:`installation_configuration_notificaties_api`)
```

Could you also add a line to state that extra configuration steps (one to configure external services and to configure notificaties) were added to configure notifications (and refer to `installation_configuration_cli`)?

* [:open-api-framework:`59`] Remove ``django.contrib.sites`` dependency

.. warning::

>     To save the domain of the application you have to declare the env setting ``SITE_DOMAIN``

Reviewer: ```suggestion
    To save the domain of the application you have to declare the environment variable ``SITE_DOMAIN`` (see :ref:`installation_env_config` > Optional for more information)
```
